packages/components/src/modal/style.scss

```
@@ -146,4 +146,25 @@
 		outline: 2px solid transparent;
 		outline-offset: -2px;
 	}
+
+	@include break-small() {
+		&.has-width-small,
+		.has-width-medium,
+		.has-width-large {
```

I believe that the medium and large rules are missing the `&.` in front

```suggestion
		&.has-width-small,
		&.has-width-medium,
		&.has-width-large {
```